lib/src/resultset-table/ResultsetTable.stories.tsx

```diff
@@ -409,5 +410,46 @@ export const DropdownAction = ResultsetActionsCellDropdown.bind({});
 DropdownAction.play = async ({ canvasElement }) => {
   const canvas = within(canvasElement);
   const dropdown = canvas.getAllByRole("button")[5];
-  await userEvent.click(dropdown);
+  userEvent.click(dropdown);
 };
+
+const columnsCheckbox = [
+  { displayValue: "Id", isSortable: true },
+  { displayValue: "Checkbox", isSortable: false },
+  { displayValue: "Name", isSortable: false },
+  { displayValue: "City", isSortable: false },
+];
+
+const rowsCheckbox = [
+  [
+    { displayValue: "001", sortValue: "001" },
+    {
+      displayValue: <DxcCheckbox size="fillParent" />,
+    },
+    { displayValue: "Peter" },
+    { displayValue: "Miami" },
+  ],
+  [
+    { displayValue: "002", sortValue: "002" },
+    {
+      displayValue: <DxcCheckbox size="fillParent" />,
+    },
+    { displayValue: "Louis" },
+    { displayValue: "London" },
+  ],
+  [
+    { displayValue: "003", sortValue: "003" },
+    {
+      displayValue: <DxcCheckbox size="fillParent" />,
+    },
+    { displayValue: "Lana" },
+    { displayValue: "Amsterdam" },
+  ],
+];
+
+export const ResultsetCheckbox = () => (
```

Comment:
This test does not provide anything related to this fix unless we add interactions.
It is also covered in the new functional test.

Reply:
I used it to test and forgot to remove it. Done!